src/components/Loader/Loader.types.ts

```diff
@@ -0,0 +1,11 @@
+import { OcBaseProps } from '../OcBase';
+
+export enum LoaderSize {
```

Comment:
Suggest using a type here, rather than an enum so a string may be passed in implementation. I'm going to be moving a lot of these enums in this direction. e.g:

`export type LoaderSize = 'flex' | 'large' | 'medium' | 'small' | undefined;`

Adding the flex type and using the `Matchmedia`, `Breakpoint` hooks, etc will ensure this will work with the upcoming Form context with no need for additional mapping.